WordPress/twentytwentyfive
# Add pattern: Hero - Full-width cover image, heading and short paragraph

**Description**



Fixes #87 - 

Adds pattern for **Hero - Full-width cover image, heading and short paragraph**

**Screenshots**



<img width="1728" alt="Block Patterns - Hero - Full-width cover image, heading and short paragraph" src="https://github.com/user-attachments/assets/0836c289-25f5-4731-85d8-534feca89e35">



**Testing Instructions**

- Create a page
- Add the patterns (found under 'features' or 'banners')
- Confirm pattern is in line with the [design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1795&t=3i8lVkj1XFXFEA11-1)
